### DISCUSSION TITLE
[Improvement] [UI] [User-Profiles] Site Awards text should be User Awards at My Settings page

### DISCUSSION THREAD

**Discussion by Alexdatadestroyer** [RetroAchievements/RAWeb#1961] [Ideas]
Sometimes, we improve things, create new features and release new versions... But, at some point, we fail to fully adapt everything to today standards which render some things to stay on a legacy way.

Since when this (#937) came to reality in 2022 when V1 was still in course. The "Site Awards" and "Reorder Site Awards" texts weren't updated as it should be. This might lead to a "confusion" for some users when trying to use that feature as they will think that only Site Awards part would be organized, not with Event Awards and Game Awards altogether.

**We are on RAWeb V3.2.2 and those texts weren't updated yet:**

![](https://i.imgur.com/1ToHUJ1.png)

The obvious suggestion is to rename those texts to "User Awards" and "Reorder User Awards". That way, the user will understand that they are going to organize their badges from all those 3 groups instead of only one.

**Comment by wescopeland** [comment]
I agree this should change.

Will do some thinking but I think we actually should remove the word "Awards", due to #1714. Perhaps this should be "Reorder Badges", or something to that effect.

Up until now, there has been a conflation of the words "award" and "badge", but they are not actually the same thing. Badges are visual representations of some awards, however not all awards necessarily have to grant badges.

**Comment by luchaos** [comment]
+1 for renaming awards here.

> Badges are visual representations of some awards, however not all awards necessarily have to grant badges.

Badges are awarded, thus are synonymous? To me, they are the same.
Technically, unlocking an achievement awards its badge to the player 😅 
The noun award has no further meaning in our context imo.
(The SiteAwards table will be renamed and/or migrated into badges, too).

> not all awards necessarily have to grant badges

I think they should. Everything that can be awarded/unlocked should provide a badge. Or differently: there is nothing to award if it doesn't have a badge as a visual representation.

**Reply by wescopeland** [reply]
I agree that all awards should have a visual representation, but where I disagree is that beaten games should be shown on a user's list of game awards / badges similar to their completions and masteries.

**Reply by luchaos** [reply]
certainly, not at this time. i'm thinking of a dedicated badges page that won't hide anything from the user.

what i'm dreaming of, and i know many others do too, is a very extensive badge system which allows for much more individualized and varied badges. beaten _could_ be one of those - doesn't have to be though. I can see them remaining hidden - i just wanted to dissect the terms as I understand them. not that i think it's the only nor the best way to think of them but from a technical standpoint i think it makes sense to get rid of "award" altogether and think of everything as "badgeable" instead.

**Reply by wescopeland** [reply]
Oh yes, we're definitely in alignment on those thoughts. My reservation from above is given the current state not wanting to cheapen the value of masteries or completions. When we have a dedicated badges/awards/milestones page what you're describing certainly makes sense.

**Reply by luchaos** [reply]
> not wanting to cheapen the value of masteries or completions

agreed! it wouldn't fit with the current presentation. i definitely approve the current implementation of beaten games in that regard.

**Comment by Alexdatadestroyer** [comment]
> Everything that can be awarded/unlocked should provide a badge.
>
> **-- luchaos**

So, this means that Beaten Games would provide a badge at some point like was being discussed on #1829?

**Reply by wescopeland** [reply]
#1829 discusses adding badges within the confines/design of the existing badge system. There is fairly staunch opposition to this.

luchaos and I were discussing them being added to an entirely overhauled badge system that rolls out at some point in the future. Seeing them have a place in an overhauled badge/award system is definitely conceivable.
